packages/validator/src/decoder/uri.js

```diff
@@ -4,7 +4,7 @@ import { Failure } from '../error.js'
 /**
  * @template {`${string}:`} Protocol
  * @param {unknown} input
- * @param {{protocol?: Protocol}} [options]
+ * @param {{protocol?: Protocol}} options
```

Comment:
Options should be optional

```suggestion
 * @param {{protocol?: Protocol}} [options]
```